docs/DEVELOPER_GUIDE.md

```diff
@@ -4,12 +4,12 @@
 
 The following tools are required for doing development on Karpenter.
 
-| Package                                                            | Version  | Install             |
-| ------------------------------------------------------------------ | -------- | ------------------- |
-| [go](https://golang.org/dl/)                                       | v1.15.3+ |                     |
-| [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) |          |                     |
-| [helm](https://helm.sh/docs/intro/install/)                        |          | `brew install helm` |
-| Other tools                                                        |          | `make toolchain`    |
+| Package                                                            | Version  | Install                |
+| ------------------------------------------------------------------ | -------- | ---------------------- |
+| [go](https://golang.org/dl/)                                       | v1.15.3+ | `brew install go`      |
```

Comment:
I think we should have instructions for other platforms as well

Reply:
Let's tackle this in a followon.